src/main/resources/mustache-templates/layout/header.mustache

```diff
@@ -7,14 +7,15 @@
 <meta name="viewport"
 	content="width=device-width, initial-scale=1, shrink-to-fit=no">
   <link rel="stylesheet" type="text/css" href="{{req.contextPath}}/webjars/bootstrap/css/bootstrap.min.css" />
- <link rel="stylesheet" type="text/css" href="{{req.contextPath}}/webjars/font-awesome/css/all.min.css" />
- <link rel="stylesheet" href="{{req.contextPath}}/assets/css/main.css" />
+  <link rel="stylesheet" type="text/css" href="{{req.contextPath}}/webjars/font-awesome/css/all.min.css" />
+  <link rel="stylesheet" href="{{req.contextPath}}/assets/css/main.css" />
+  <link rel="stylesheet" type="text/css" href="{{req.contextPath}}/webjars/bootstrap-vue/bootstrap-vue.css" />
```

Comment:
You probably want to put `main.css` after `bootstrap-vue.css` to facilitate overriding the bootstrap-vue styles.